tests/test_models.py

```diff
@@ -391,3 +390,74 @@ def test_cct():
     f = model(img_3channels_224)
     assert f.shape == (4, 1000)
     del model
+
+
+def test_visformer():
+
+    model = MODEL_REGISTRY.get("Visformer_S")(224, 1000)
+    out = model(img_3channels_224)
+    assert out.shape == (4, 1000)
+    del model
+
+    model = MODEL_REGISTRY.get("Visformer_Ti")(224, 1000)
+    out = model(img_3channels_224)
+    assert out.shape == (4, 1000)
+    del model
+
+    model = MODEL_REGISTRY.get("VisformerV2_S")(224, 1000)
+    out = model(img_3channels_224)
+    assert out.shape == (4, 1000)
+    del model
+
+    model = MODEL_REGISTRY.get("VisformerV2_Ti")(224, 1000)
+    out = model(img_3channels_224)
+    assert out.shape == (4, 1000)
+    del model
+
+
+def test_dpt():
+    img = torch.randn(4, 3, 384, 384)
+    model = MODEL_REGISTRY.get("DPTDepth")(
+        "vitb16",
+        enable_attention_hooks=True,
+    )
+    del model
```

Comment:
```suggestion
    del model
    
```

Reply:
@aditya-agrawal-30502  
Umm i did not get you